app/models/waste_carriers_engine/order.rb

```diff
@@ -25,19 +25,12 @@ class Order
     field :manualOrder, as: :manual_order,           type: String
     field :order_item_reference,                     type: String
 
+    # TODO: Move to a service
```

Comment:
If the intention is to move this to an `OrderRenewalService` (or something like that) would it be better to do that refactoring first and then add your new service?

As is I'm just not sure I like the idea that if we never get round to this `TODO` we'll have `Order` create the order with `new_order`, set the order items as if its a renewal, but then `OrderAdditionalCardsService` take that order and simply overwrites the array. Plus we're creating orders in 2 different ways.

Reply:
This service do not use the code in `new_order` so I have left the TODO for the next person touching it. We should get there naturally when we work out the front end flow.

Reply:
🤦‍♂ Yep, was pointed out to me I was misreading the changes!